images/rabbitmq/Dockerfile

```diff
@@ -3,9 +3,20 @@ FROM rabbitmq:3-management
 ARG LAGOON_VERSION
 ENV LAGOON_VERSION=$LAGOON_VERSION
 
-COPY rabbitmq_delayed_message_exchange-3.7.0.ez /plugins
+ENV RABBITMQ_DEFAULT_USER=guest \
+    RABBITMQ_DEFAULT_PASS="guest"\
+    RABBITMQ_DEFAULT_HA_PATTERN='^lagoon-'
 
+
+COPY rabbitmq_delayed_message_exchange-3.7.0.ez /plugins
 RUN rabbitmq-plugins enable --offline rabbitmq_delayed_message_exchange;
 
-ENV RABBITMQ_DEFAULT_USER=guest \
-    RABBITMQ_DEFAULT_PASS="guest"
+# Copy startup schema with vhost, users, permissions and policies
+COPY definitions.json /etc/rabbitmq/definitions.json
+RUN chgrp 0 /etc/rabbitmq/definitions.json; chmod g+rw /etc/rabbitmq/definitions.json
+
+# Copy a custom entrypoint
+ADD cluster-rabbit.sh /
```

Comment:
let's use `COPY` here as we use everywhere else.

Reply:
Done.